packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1089,7 +1089,6 @@ const toSphinxTransactionEstimatedGas = (
     from: response.from,
     data: response.data,
     gasLimit: response.gasLimit.toString(),
-    gasPrice: response.gasPrice.toString(),
```

Comment:
Removed the gas price here because it isn't useful for estimating fees on the website. If we need this on the website, we'll just populate it with the current gas price on the network.